f600233d1b154d507b782f84604bcb12/README.md

```diff
@@ -16,12 +16,3 @@ If you find yourself unable to import your repository due to the presence of fil
   4. Run `git add -A`
   5. Run `git commit`
   6. Run `git push`
-
- **Notes on dependencies**
-  1. Dependencies will not show up in the list of changes but will be exported/imported
-  2. It is your responsibility to resolve the dependencies before installing an application. ServiceNow source control will not manage these for you. In case you installed an application before installing its dependencies:
-   2.1 Delete the application
-   2.2 Activate/install all required dependencies
-   2.3 Re-import the application from source control
-   Currently listed dependencies:
-   * System Import Sets
```

Comment:
I'm guessing this wasn't intentional. Maybe your PDI doesn't like this section of the README, haha.

Reply:
Dang.  No not intentional.

Reply:
Fixed in cc12fd9d221430f24f14f60efec0b50076026626